sre-server/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+WD="$(dirname $0)/../certs"
+HOST="localhost"
+PORT=8443
+
+CURL="curl -v --cert ${WD}/client.crt --key ${WD}/client.key --cacert ${WD}/ca.crt -H \"Content-Type: application/json\""
+
+PASSED () {
+    echo -e "\033[1m\033[42mPASSED\033[0m"
+}
+FAILED () {
+    echo -e "\033[1m\033[41mFAILED\033[0m"
+}
```

Comment:
Should this exit on failure?

Reply:
I don't think that exit after first failed test is good practice. Usually test packages always run all tests.

I added exit code and test.sh will exit with code 1 when any test will fail.